ramls/eholdings.raml

```diff
@@ -7,6 +7,109 @@ documentation:
   - title: mod-kb-ebsco (category)
     content: Implements the eholdings interface using the EBSCO kb as a backend.
 
+traits:
+  indexable:
+    queryParameters: 
+      q:
+        displayName: Query
+        type: string
+        description: Text entered in search field
+        example: Basket Weaving
+        required: true
+      page:
+        displayName: Page
+        type: number
```

Comment:
If there is a way to provide a min value here for page offset, we should specify that as 1, else RM API will fail

Reply:
Reviewing the specification docs revealed that there is indeed a way to restrict the available numbers using `minimum` and `maximum`. -->  https://github.com/raml-org/raml-spec/blob/master/versions/raml-10/raml-10.md/#number
I have gone ahead and updated with those changes.